frontend/src/components/common/icon/icon.tsx

```diff
@@ -32,7 +32,11 @@ type Props = {
 const Icon: FC<Props> = ({ name, className }) => {
   const SelectedIcon = iconNameToIcon[name];
 
-  return <SelectedIcon className={getValidClasses(styles.icon, className)} />;
+  return (
+    <SelectedIcon
+      className={getValidClasses(styles.icon, className ?? styles[name])}
```

Comment:
we need to do it with the class name prop
```suggestion
      className={getValidClasses(styles.icon, className)}
```